src/checkers/cwe_248.mli

```diff
@@ -0,0 +1,11 @@
+(** This module implements a check for CWE-248 (Uncaught Exception)
+An uncaught exception may lead to a crash and subsequentially to other unintended behavior.
+See https://cwe.mitre.org/data/definitions/248.html for detailed description.
+
+Right now we search for exception throws that are reachable in the callgraph without
+touching a function that contains a catch block. We do not check whether a catch block
+can actually catch the thrown exceptions, thus we generate some false negatives. **)
+val name : string
+val version : string
+
+val check_cwe : Bap.Std.program Bap.Std.term -> Bap.Std.project -> Bap.Std.word Bap.Std.Tid.Map.t -> string list list -> unit
```

Comment:
the Bap.Std interface is designed to be opened (it tries not to pollute the namespace). So once it is open you can easily write the interface in a much more concise and readable way: `program term -> project -> word Tid.Map.t -> string list list -> unit`